js/main.js

```diff
@@ -100,3 +125,23 @@ document.addEventListener('DOMContentLoaded', function (event) {
   }
   showView(data.view);
 });
+
+// Editing
+$ul.addEventListener('click', edit);
+
+function edit(e) {
+  var targetid = e.target.getAttribute('data-entry-id');
+  if (e.target.nodeName === 'I') {
+    for (let i = 0; i < data.entries.length; i++) {
+      if (data.entries[i].entryId === parseInt(targetid)) {
+        data.editing = data.entries[i];
+      }
+    }
+  }
+  entryformView();
+
+  $heading.childNodes[0].nodeValue = 'Edit Entry';
+  $title.value = data.editing.title;
+  $photoURL.value = data.editing.photoURL;
```

Comment:
make sure to also update the `src` of the image tag so that the image shows on the edit entry form.

Reply:
![image](https://user-images.githubusercontent.com/94674653/180347775-f6bcd9df-c5ef-4049-8198-5f77dcb20ed9.png)